index.html

```diff
@@ -77,110 +118,118 @@
-                            <picture>
-                                <source srcset="https://assets.csh.rit.edu/pubsite/projects/schedulemaker.webp" type="image/webp" />
-                                <source srcset="https://assets.csh.rit.edu/pubsite/projects/schedulemaker.jpg" type="image/jpeg" />
-                                <img class="rounded"  src="https://assets.csh.rit.edu/pubsite/projects/schedulemaker.jpg" alt="The logo for CSH Schedule Maker" />
-                            </picture>
-                        </a>
-                    </div>
-                    <div class="col-3">
-                        <picture>
-                            <source srcset="https://assets.csh.rit.edu/pubsite/projects/arcade.webp" type="image/webp" />
-                            <source srcset="https://assets.csh.rit.edu/pubsite/projects/arcade.jpg" type="image/jpeg" />
-                            <img class="rounded"  src="https://assets.csh.rit.edu/pubsite/projects/arcade.jpg" alt="Two custom arcade machines in the CSH library" />
-                        </picture>
-                    </div>
-                    <div class="col-3">
-                        <a href="https://github.com/ramzallan/tonic" target="_blank">
-                            <picture>
-                                <source srcset="https://assets.csh.rit.edu/pubsite/projects/drink.webp" type="image/webp" />
-                                <source srcset="https://assets.csh.rit.edu/pubsite/projects/drink.jpg" type="image/jpeg" />
-                                <img class="rounded"  src="https://assets.csh.rit.edu/pubsite/projects/drink.jpg" alt="Two of CSH's networked vending machines, Drink and Snack.">
-                            </picture>
-                        </a>
-                    </div>
-                    <div class="col-3">
-                        <a href="https://github.com/nfatkhiyev/Harold" target="_blank">
-                            <picture>
-                                <source srcset="https://assets.csh.rit.edu/pubsite/projects/harold.webp" type="image/webp" />
-                                <source srcset="https://assets.csh.rit.edu/pubsite/projects/harold.jpg" type="image/jpeg" />
-                                <img class="rounded"  src="https://assets.csh.rit.edu/pubsite/projects/harold.jpg" alt="Harold, a box with a sensor on it, that plays members' music" />
-                            </picture>
-                        </a>
-                    </div>
-                </div>
+            <div class="stat_card">
+                <p class="stat_big">3<sup style="font-size: 0.5em;">rd</sup></p>
+                <p class="stat_tag">Greatest Hack</p>
+                <p class="stat_desc">Of all time (as ranked by PCMAG in 2008) was the creation of CSH's internet-connected drink machines.</p>
             </div>
-        </div>
+            <div class="stat_card">
+                <p class="stat_big">1388</p>
```

Comment:
Is the plan to eventually make this a live number?

Reply:
Yes, that number was pulled manually from ldap. I'm not quite ready to write that part, I want to get some help to make sure that I'm doing it in a secure manner.

Reply:
Makes sense. Odds are we can give the CI in OKD a service account it can use to make some queries during build.